src/attributes/components/AttributeProperties/AttributeProperties.tsx

```diff
@@ -25,88 +26,126 @@ const AttributeProperties: React.FC<AttributePropertiesProps> = ({
-        <TextField
+      <CardContent>
+        {/* <Typography variant="subtitle1">
+          <FormattedMessage
+            defaultMessage="General Properties"
+            description="attribute general properties section"
+            id="attributePropertiesGeneralSectionTitle"
+          />
+        </Typography>
+        <Hr />
+        <CardSpacer />
+        <ControlledSwitch
+          name={"" as keyof AttributePageFormData}
+          checked={false}
+          disabled={disabled}
+          label={
+            <>
+              <FormattedMessage
+                defaultMessage="Variant Attribute"
+                description="attribute is variant-only"
+                id="attributePropertiesVariantOnly"
+              />
+              <Typography variant="caption">
+                <FormattedMessage
+                  defaultMessage="If enabled, you'll be able to use this attribute to create product variants"
+                  id="attributePropertiesVariantOnlyHelperText"
+                />
+              </Typography>
+            </>
+          }
+          onChange={onChange}
+        /> */}
+
+        <Typography variant="subtitle1">
+          <FormattedMessage
+            defaultMessage="Storefront Properties"
+            description="attribute properties regarding storefront"
+            id="attributePropertiesStorefront"
+          />
+        </Typography>
+        <Hr />
+        <ControlledSwitch
+          name={"filterableInStorefront" as keyof AttributePageFormData}
+          checked={data.filterableInStorefront}
           disabled={disabled}
-          error={!!errors.storefrontSearchPosition}
-          fullWidth
-          helperText={errors.storefrontSearchPosition}
-          name={"storefrontSearchPosition" as keyof AttributePageFormData}
-          label={i18n.t("Position in faceted navigation")}
-          value={data.storefrontSearchPosition}
+          label={intl.formatMessage({
+            defaultMessage: "Use in faceted navigation",
+            description: "attribute is filterable in storefront",
+            id: "attributePropertiesUseInFacetedNavigation"
+          })}
           onChange={onChange}
         />
-      )}
-      <FormSpacer />
-      <ControlledSwitch
-        name={"visibleInStorefront" as keyof AttributePageFormData}
-        checked={data.visibleInStorefront}
-        disabled={disabled}
-        label={i18n.t("Visible on Product Page in Storefront", {
-          context: "attribute"
-        })}
-        onChange={onChange}
-      />
-      <CardSpacer />
+        {data.filterableInStorefront && (
+          <TextField
+            disabled={disabled}
+            error={!!errors.storefrontSearchPosition}
+            fullWidth
+            helperText={errors.storefrontSearchPosition}
+            name={"storefrontSearchPosition" as keyof AttributePageFormData}
+            label={intl.formatMessage({
+              defaultMessage: "Position in faceted navigation",
```

Comment:
Not sure everyone will understand what "faceted navigation" is. I know it was like that before your PR, we should track this as a separate issue.